packages/portal/src/mixins/logEvent.js

```diff
@@ -15,7 +15,7 @@ export default {
       return !!(
         this.$features?.eventLogging &&
         this.eventToLog &&
-        !this.eventLogged &&
+        !this.eventLogged && // QUESTION: is this giving false positives w/ multi-select...?
```

Comment:
Partially yes, but there's another issue that impacts multi-select logging especiallyi around lolgging events where some items in the selection have already had an event logged for the same action. Specifically that's noted in server-middleware/api/events.

Trying to keep `eventLogged` as a data property doesn't make too much sense if the mixin is used in the buttons part of the selection toolbar where it  may be responsible for logging various unique events. Unique because each event has a different combination of objects.

Reply:
Addressed on a separate PR: https://github.com/europeana/portal.js/pull/2668